crates/gitbutler-tauri/src/open.rs

```diff
@@ -0,0 +1,24 @@
+use crate::error::Error;
+use tracing::instrument;
+
+pub fn open_that(path: String) {
+    // `open` required to run in separate thread, to avoid blocking on some
+    // platforms (eg Fedora38 blocks)
+    std::thread::spawn(|| {
+        for mut cmd in open::commands(path) {
```

Comment:
It's very interesting that this is needed as I always thought that `tauri` is already using `open-rs` under the hood. Thus I would have hoped it works just as well there as it can work here.

Reply:
Yeah it is, but if you use the "open-shell" feature from tauri, it also packs the `xdg-open` binary from the build env, which I'm trying to avoid.

This is still very much WIP, might need to whitelist `xdg-open` as a binary we can call from the users system as well